google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/AdminClientSettings.java

```diff
@@ -25,29 +25,21 @@
 import io.grpc.StatusException;
 import java.io.IOException;
 import java.util.Optional;
-import org.threeten.bp.Duration;
 
 @AutoValue
 public abstract class AdminClientSettings {
-  public static final RetrySettings DEFAULT_RETRY_SETTINGS =
-      RetrySettings.newBuilder()
-          .setInitialRetryDelay(Duration.ofMillis(100))
-          .setRetryDelayMultiplier(1.3)
-          .setMaxRetryDelay(Duration.ofSeconds(60))
-          .setJittered(true)
-          .setTotalTimeout(Duration.ofMinutes(10))
-          .build();
 
   // Required parameters.
   abstract CloudRegion region();
 
-  // Optional parameters.
-  abstract Optional<RetrySettings> retrySettings();
+  abstract RetrySettings retrySettings();
```

Comment:
nit. this is an optional parameter (users dont need to set it)

Reply:
Changed the location of the comment